sulong/projects/com.oracle.truffle.llvm.parser.factories/src/com/oracle/truffle/llvm/parser/factories/BasicNodeFactory.java

```diff
@@ -1362,6 +1362,8 @@ protected LLVMExpressionNode getLLVMBuiltin(FunctionDeclaration declaration, LLV
                     return LLVMPowNodeGen.create(args[1], args[2]);
                 case "llvm.powi.f80":
                     return LLVMPowNodeGen.create(args[1], args[2]);
+                case "llvm.round.f80":
```

Comment:
I guess we can also put `llvm.round.f32` and `llvm.round.f64` here, right?

Reply:
Yep, let me know if you want me to commit that or if you want to add it when you merge